.gitignore

```diff
@@ -0,0 +1 @@
+vendor/
```

Comment:
Do we actually need this? Not that it would matter but it seems we're not using the vendor dir here (or elsewhere for that matter).

Reply:
removed